### PR TITLE
GWI tutorials

### PR DESCRIPTION
**Pull request by Vikramkulkarni01**
Hi @jmmargo  and @thecodester, 

My old PR was auto-closed post performing the fork sync. Hence, I'm creating this new one. 

please do the needful.

### CODE REVIEW

**Comment by thecodester** [comment]
You are pushing 12 document management tutorials and 1 web analytics tutorial?

Please fix some of the titles:
- "Create a Destination" is too generic -- "Create a Destination to  Google Drive" makes more sense
- Do not use "-ing" for the titles, like in "Managing Users ..." -- that is a title for documentation not for a specific action someone will do in this tutorial
- Some titles are quite long like "Create a Service Instance and then a Service Key of SAP Document Management Service, Integration Option", which could be shortened perhaps to "Create Service Instance for SAP Document Management Service" or "Create Service Instance for Integration Option" (depending on the meaning of Integration Option)

The step names are general OK, but should be very short, for example, "Create an instance of the Document Management Service, Integration Option" can be shortened to "Create instance of Document Management Service" or even "Create service instance". Also, they should all be sentence case. While you have "Create new destination" (good), there is also "Create a Service Key in SAP BTP" . Also, in that last step, why mention SAP BTP -- that entire tutorial takes place in SAP BTP, not just that step.

Also, you have formatting issues -- just add blank line before every image.

image

Also, I don't see any special tagging -- just generic BTP.  (1) Are there any special tags, you spoke with Joshua about? I see in the community a tag for Document Management service? (2) Some of the tags are inconsistent -- this one has cockpit tag and another tutorial that also used the cockpit does not.

image

Please update. I will look at the rest of the contents later today.

**Comment by Vikramkulkarni01** [comment]
Hi

> You are pushing 12 document management tutorials and 1 web analytics tutorial?
> 
> Please fix some of the titles:
> 
> * "Create a Destination" is too generic -- "Create a Destination to  Google Drive" makes more sense
> * Do not use "-ing" for the titles, like in "Managing Users ..." -- that is a title for documentation not for a specific action someone will do in this tutorial
> * Some titles are quite long like "Create a Service Instance and then a Service Key of SAP Document Management Service, Integration Option", which could be shortened perhaps to "Create Service Instance for SAP Document Management Service" or "Create Service Instance for Integration Option" (depending on the meaning of Integration Option)
> 
> The step names are general OK, but should be very short, for example, "Create an instance of the Document Management Service, Integration Option" can be shortened to "Create instance of Document Management Service" or even "Create service instance". Also, they should all be sentence case. While you have "Create new destination" (good), there is also "Create a Service Key in SAP BTP" . Also, in that last step, why mention SAP BTP -- that entire tutorial takes place in SAP BTP, not just that step.
> 
> Also, you have formatting issues -- just add blank line before every image.
> 
> image
> 
> Also, I don't see any special tagging -- just generic BTP. (1) Are there any special tags, you spoke with Joshua about? I see in the community a tag for Document Management service? (2) Some of the tags are inconsistent -- this one has cockpit tag and another tutorial that also used the cockpit does not.
> 
> image
> 
> Please update. I will look at the rest of the contents later today.

Hi @thecodester , 

Thanks for your feedback. 

Please find my response below:

1. I'm pushing only 12 tutorials and I'm not sure how web analytics tutorial is being picked. However, I don't want web analytics tutorial to appear as this is service is deprecated. I need only 12 GWI tutorials to be moved to prod. 
2. Tags have been added as per the tutorial specific and used special tags like document management service and s/4hana cloud
3. Titles have been fixed as per your recommendation
4. formatting issues have been fixed. However, while in preview it looks fine but once the build runs, I can see some differences.
5. All images now added with a line prior to it. 

Please do the needful.

File: tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ - You've generated a JSON Web Token (JWT) that serves as your authorization for making API calls. For more information, see the tutorial [Generating the JSON Web Token (JWT)](btp-sdm-gwi-onbrepo-jwt-token).
+ - You've the admin permission for Document Management Service, Integration Option via the **`SDM_Admin`** scope.
+ - You've installed the [Postman](https://www.postman.com/downloads/) app.
+
+### You will learn
+  - How to onboard a repository using API.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Get URL value from the service key)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md:24]: No need for word "value" here

File: tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+- You've an account in the Google Workspace.
+- You've necessary privileges for SAP S/4HANA Cloud, public edition.
+
+### You will learn
+
+- How to export data from SAP UI (User Interface) tables.
+
+---
+[ACCORDION-BEGIN [Step 1: ](Access Your Application )]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md:21]: Sentence case

File: tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Log on to your S/4HANA Cloud, public edition. Open the **Manage Journal Entries** or **Purchase Orders** or any other applications that is offering a table-based export.
+
+  !![Accessing System](LogOntoS4HANACloud.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Exporting Table Details)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md:30]: Sentence case

File: tutorials/btp-sdm-gwi-s4hana-maintain-businessroles/btp-sdm-gwi-s4hana-maintain-businessroles.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+3. Choose **Save**.
+
+    !![Save](Save.png)
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 5: ](Knowledge check)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hana-maintain-businessroles/btp-sdm-gwi-s4hana-maintain-businessroles.md:126]: All the other ones you said "Test yourself"

File: tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+## Prerequisites
+ - You've an admin access to SAP S/4HANA Cloud, public edition.
+
+### You will learn
+- How to maintain protection allow list on SAP S/4HANA Cloud, public edition.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Log On to the SAP S/4HANA cloud application)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:20]: Why not "Log into SAP S/4HANA Cloud"?

File: tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  Log on to SAP S/4HANA Cloud, public edition and search for the **`Maintain Protection Allowlists`** app from the SAP Fiori Launchpad.
+  !![LogonScreenApp](MaintainProtectionAllowlistApp.png)
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 2: ](Creating trusted network zone)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:28]: No ing verbs

File: tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  Navigate to the **Trusted Network Zones** tab and click **(+) Create**.
+
+   !![TrustedNetworkzone](TrustedNetworkzone.png)
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Managing the details)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:37]: No ing verbs

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+ - You've copied the `clientid`, `clientsecret`, and `tokenurl` parameters generated in the same subaccount of Cloud Foundry environment where Document Management Service, Integration Option instance is created.
+
+
+### You will learn
+ - How to use the existing communication scenario `SAP_COM_0190`
+ - How to add details from the service key generated from SAP BTP to SAP S/4HANA cloud, public edition
+ - How to add the details that are captured from SAP BTP in SAP S/4HANA Cloud, public edition
+---
+
+[ACCORDION-BEGIN [Step 1: ](Create Communication Arrangements)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:25]: "communication arrangement" is lower case (except when referring to UI) -- sentence case

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+
+    !![CreateNewComApp](NewCommApp_Create.png)
+
+>**Results:** Communication Arrangement has been created, and you're directed to the communication arrangement page to configure further settings.
+  Stay on the same page and proceed to the next step.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Creating Communication System)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:49]: Same

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+9. **Save** the Communication System.
+
+    !![SavetheCommsystem](SavetheCommsystem.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Enter Additional Properties in the Communication Arrangements)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:104]: Just say "Enter addition properties"

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
+---
+title: Trust Configuration between SAP S/4HANA Cloud, public edition and SAP BTP Subaccount
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:2]: No verb here

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+ - You've access to the SAP S/4HANA Cloud, public edition system.
+ - You've access to the Google Workspace Account. For more information, see [Configure Service Account in Google Cloud Platform](https://help.sap.com/docs/IRPA/1154f48dd7ab430ea52badeb4359e4b4/40baf1a31fad4e86892795f7fe59d971.html).
+
+## You will learn
+  - How to configure trust between SAP S/4HANA Cloud, public edition and SAP BTP system.
+  - How to manage trust configurations between SAP S/4HANA Cloud, public edition and SAP BTP.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Downloading SAML2.0 Metadata from SAP S/4HANA Cloud)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:23]: No ing verbs

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+    !![ShowDetailsIssuer2](screenshots.png/ShowDetailsIssuer2.png)
+
+    !![ShowDetailsIssuer3](screenshots.png/ShowDetailsIssuer3.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Add Users in the SAP BTP cockpit)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:64]: Sentence case (lower case users)

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+    ```JSON
+      <md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:URI" Location="https://example.com"index="1"/>
+    ```
+    !![AssertionConsumerService](screenshots.png/AssertionConsumerService.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Test Yourself)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:101]: lower case Y

File: tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+
+### You will learn
+  - How to add users to the Google Workspace admin console.
+  - How to manage user information.
+
+> **Tip**: You can skip this tutorial if you have already created a user in the Google Workspace console. If it has not yet been created, follow these steps:
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Adding User)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md:24]: No ing verbs

File: tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+Log in to your **Google Workspace** account using *`admin.google.com`* and click on **Add a User**.
+
+  !![Adding User](Add_User.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Manage User Information)]
```
Comment (thecodester) [inline tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md:32]: Sentence case

File: tutorials/btp-sdm-gwi-create-serviceinstance/btp-sdm-gwi-create-serviceinstance.md [hunk 17]
```diff
@@ -0,0 +1,128 @@
+---
+title: Create a Service Instance and Service Key for SAP Document Management Service, Integration Option
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-create-serviceinstance/btp-sdm-gwi-create-serviceinstance.md:2]: Hi @thecodester, 
Shortened a bit. But I'm wondering about the consistency in using brand names.

File: tutorials/btp-sdm-gwi-create-serviceinstance/btp-sdm-gwi-create-serviceinstance.md [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+
+You'll export table data from SAP S/4HANA Cloud, public edition to Google Sheets. You'll achieve this scenario by integrating SAP S/4HANA with Google Workspace using the SAP Document Management Service.
+
+Here SAP S/4HANA acts as a source system and Google Drive acts as a target system. You'll also create trust between these systems via SAP BTP cockpit.
+
+  ![Use Case](UseCase_Architecture.png)
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Create an instance of the Document Management Service, Integration Option)]
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-create-serviceinstance/btp-sdm-gwi-create-serviceinstance.md:30]: shortened

File: tutorials/btp-sdm-gwi-create-destination/btp-sdm-gwi-create-destination.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+ - What are some of the additional properties you need to add while configuring the destination?
+ - How to specify the Google parameters that are required to create a destination
+
+### Information
+  Destinations are key building blocks in SAP BTP, and are used to define connections for outbound communication from your application to remote systems. These remote systems can be on-premises or in the cloud. A destination has a name, a URL, authentication details, and some other configuration details.
+
+---
+
+
+[ACCORDION-BEGIN [Step 1: ](Downloading the public or private key pair JSON file from Google Cloud console)]
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-create-destination/btp-sdm-gwi-create-destination.md:30]: incorporated

File: tutorials/btp-sdm-gwi-create-destination/btp-sdm-gwi-create-destination.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+  When you specify a URL with the HTTPS scheme, a checkbox Use default **JDK truststore** will appear. Ensure that this is checked.
+
+  !![HTTPDestination](NewDestinatioFormat.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Check Connection)]
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-create-destination/btp-sdm-gwi-create-destination.md:107]: corrected

File: tutorials/btp-sdm-gwi-crossdomain-mapping/btp-sdm-gwi-crossdomain-mapping.md [hunk 21]
```diff
@@ -0,0 +1,81 @@
+---
+title: Manage Cross-Domain Mapping
+description: The steps mentioned in the tutorials are required if the domain of your email account and Google Workspace is different. In such cases, we need to map your system user account to the Google Workspace domain. For example, if you're using the domain abc.com (jhon.thane1@abc.com) and have a Google Workspace account with the domain abcworkspace.com (jhon.thane1@abcworkspace.com), you need to maintain cross-domain mapping.
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-crossdomain-mapping/btp-sdm-gwi-crossdomain-mapping.md:3]: Now used only a sentence. The rest information has been moved to the introduction section.

File: tutorials/btp-sdm-gwi-crossdomain-mapping/btp-sdm-gwi-crossdomain-mapping.md [hunk 22]
```diff
@@ -0,0 +1,81 @@
...
+|  Property Name     | Value
+|  :------------- | :-------------
+|  **`google.default_domain`** | **`Enter the domain.`** For example, `abcworkspace.com`   
+
+### You will learn
+  - How do you manage cross domain mapping at the service instance level
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Get URL value from the service key)]
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-crossdomain-mapping/btp-sdm-gwi-crossdomain-mapping.md:29]: Done

File: tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+    >**Note** Please modify the details according to your needs. It is not necessary to follow the same code snippet. A few details, such as display name, and destination names, may change. It is important that you read the Prerequisites again.
+
+    !![EnterCodeSnippet](EnterCodeSnippet in the Body.png)
+
+    Click **Send**.
+
+    >**Results:** When a successful response is received, the status is `200 OK`.
+
+
+    **Results**
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md:76]: Yes, we need this. Just to help the first-time users. 
We have a plan to remove this in our incremental updates.

File: tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md [hunk 24]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    ```
+
+    >**Note** Please modify the details according to your needs. It is not necessary to follow the same code snippet. A few details, such as display name, and destination names, may change. It is important that you read the Prerequisites again.
+
+    !![EnterCodeSnippet](EnterCodeSnippet in the Body.png)
+
+    Click **Send**.
+
+    >**Results:** When a successful response is received, the status is `200 OK`.
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md:73]: Just to highlight it. However, I changed it to a number now.

File: tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md [hunk 25]
```diff
@@ -0,0 +1,52 @@
+---
+title: Testing Table Export Using SAP UI5 Application
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md:2]: Corrected it.

File: tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md:21]: Corrected it.

File: tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md:30]: Changed

File: tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:37]: changed

[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:37]: changed.

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:25]: Sure. Changed it to Create communication arrangements

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:49]: Sure. Changed it to Create communication system

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:2]: Added verb now

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:64]: corrected

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md [hunk 26]
```diff
@@ -0,0 +1,109 @@
...
+    !![AssignRoleCollections](screenshots.png/AssignRoleCollections.png)
+
+3. Assign the user role collection of the **SAP Document Management Service, Integration Option** (For example, `SDM_roles` or the role collection that you created) which is defined in the subaccount. For more information, see the 3rd step in this tutorial [Create a Service Instance and then a Service Key of SAP Document Management Service, Integration Option](btp-sdm-gwi-create-serviceinstance).
+
+    !![SDM_RoleCollections](screenshots.png/SDM_RoleCollections.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Download SAML Metadata from SAP BTP Cockpit)]
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:85]: corrected

File: tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md:24]: combined

File: tutorials/cp-webanalytics-setup/cp-webanalytics-setup.md [hunk 27]
```diff
@@ -0,0 +1,135 @@
+---
```
[the remark this answers is not in the document]
Comment (Vikramkulkarni01) [reply tutorials/cp-webanalytics-setup/cp-webanalytics-setup.md:1]: deleted

File: tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md (hunk 1 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-onboarding-repo/btp-sdm-gwi-onboarding-repo.md:24]: corrected

File: tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md (hunk 2 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-export-testing/btp-sdm-gwi-s4hana-export-testing.md:21]: corrected

File: tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md (hunk 5 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:20]: changed

File: tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md (hunk 7 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-protection-allowlist/btp-sdm-gwi-s4hana-protection-allowlist.md:37]: corrected

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md (hunk 9 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:49]: incorporated

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md (hunk 11 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:2]: added it now

File: tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md (hunk 14 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-export/btp-sdm-gwi-s4hanacloud-export.md:101]: corrected

File: tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md (hunk 16 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-user-management/btp-sdm-gwi-user-management.md:32]: combined step1 and step 2

File: tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md (hunk 10 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md:104]: corrected

File: tutorials/btp-sdm-gwi-s4hana-maintain-businessroles/btp-sdm-gwi-s4hana-maintain-businessroles.md (hunk 4 above)
Comment (Vikramkulkarni01) [reply tutorials/btp-sdm-gwi-s4hana-maintain-businessroles/btp-sdm-gwi-s4hana-maintain-businessroles.md:126]: corrected

**Comment by Vikramkulkarni01** [comment]
Hi @thecodester and @jmmargo, 

Could you please help us in merging the PR asap?
As I'm receiving speedup requests from the Dev team. I truly appreciate your help. 

Regards, 
Vikram

**Pull request merged**